### PR TITLE
fix: Pages Functions custom _routes.json not being used

### PR DESCRIPTION
**Pull request by jahands**

### CODE REVIEW

File: packages/wrangler/src/pages/publish.tsx [hunk 1]
```diff
@@ -348,6 +346,7 @@ export const Handler = async ({
 			}
 
 			_routes = JSON.stringify(optimizeRoutesJSONSpec(advancedModeRoutes));
+			formData.append("_routes.json", new File([_routes], "_routes.json"));
```
Comment (jrf0110) [inline packages/wrangler/src/pages/publish.tsx:349]: Isn't `_routes` undefined here?

Comment (jahands) [reply packages/wrangler/src/pages/publish.tsx:349]: No, it's defined a ways up and then set right above this on line 348.
I tested this to be sure

**Review by CarmenPopoviciu — APPROVED** [review]
LGTM!

File: packages/wrangler/src/pages/publish.tsx (hunk 1 above)
Comment (jrf0110) [reply packages/wrangler/src/pages/publish.tsx:349]: oh wow I'm a dumb dumb totally missed line 348 😅

**Pull request merged**